### PR TITLE
Fix error format and info for get_test.go

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:
Fix error format and info for get_test.go
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/kubectl/cmd/get_test.go [hunk 1]
```diff
@@ -110,7 +110,7 @@ func testComponentStatusData() *api.ComponentStatusList {
 
 	unknown := api.ComponentStatus{
 		Conditions: []api.ComponentCondition{
-			{Type: api.ComponentHealthy, Status: api.ConditionUnknown, Message: "", Error: "fizzbuzz error"},
+			{Type: api.ComponentHealthy, Status: api.ConditionUnknown, Message: "", Error: "f)izzbuzz error"},
```
Comment (xiangpengzhao) [inline pkg/kubectl/cmd/get_test.go]: Why adding a `)` here :question:

Comment (zhangxiaoyu-zidif) [inline pkg/kubectl/cmd/get_test.go]: oh, my bad... Thank you.=)

**Comment by mengqiy** [comment]
/assign @pwittrock 
Assign to you for approval.

**Comment by zhangxiaoyu-zidif** [comment]
@adohe For approval

**Comment by zhangxiaoyu-zidif** [comment]
@pwittrock 
For approval

**Comment by zhangxiaoyu-zidif** [comment]
@fabianofranz 
PTAL

**Pull request merged**